### PR TITLE
Add a "press release" for our performance in 2018

### PR DESCRIPTION
**Pull request by antoinealb**
This PR adds a small article about how we finished second in 2018.

We will write something more involved to describe the robots after the french cup.

### CODE REVIEW

File: _posts/2018-04-21-cvra-second-swisseurobot.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The first day was difficult for the CVRA.
+The robots had reliability issues, and it was hard for us to pick up the cubes.
+We finished the day at the 8th position, with a total of 322 points.
+
+However, we spent the night working and the next day was much better for us.
+The first game of Saturday we scored 266 points, almost doubling our total!
+We climbed up the ladder after this and were qualified for the finals.
+
+The finals were quite stressful for us.
+We lost the first game, which meant that any further lost match would mean the end of the competition.
```
Comment (SyrianSpock) [inline _posts/2018-04-21-cvra-second-swisseurobot.md:20]: lost by 2 points :sob:

File: _posts/2018-04-21-cvra-second-swisseurobot.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Fortunately, everything went well, and we went to the final game, where we lost against Happy Social Robot.
+
+
+![CVRA team with our two robots, Order & Chaos](/images/posts/swisseurobot2018/team.jpg)
+
+As we are in the first three Swiss teams, we will represent Switzerland in the European finals.
```
Comment (SyrianSpock) [inline _posts/2018-04-21-cvra-second-swisseurobot.md]: European? World?

**Review by SyrianSpock — APPROVED** [review]
Nice initiative, thanks :+1:

File: _posts/2018-04-21-cvra-second-swisseurobot.md (hunk 2 above)
Comment (antoinealb) [reply _posts/2018-04-21-cvra-second-swisseurobot.md]: I went with "Eurobot finals"

**Pull request merged**